packages/i18n-utils/src/locale-context.tsx

```diff
@@ -1,6 +1,7 @@
 import { createHigherOrderComponent } from '@wordpress/compose';
 import * as i18n from '@wordpress/i18n';
 import React, { createContext, useContext, useEffect, useState } from 'react';
+import { Locale } from './locales';
```

Comment:
When importing only type info we do this so that it doesn't impact bundle size.

```suggestion
import type { Locale } from './locales';
```